docs/ossm/ambient/README.md

```diff
@@ -0,0 +1,340 @@
+* L7 features require the deployment of `waypoint` proxies, which add a small amount of overhead for the services that utilize them.
+
+---
+
+## 2. Pre-requisites to Using Ambient Mode with OSSM 3
+
+Before installing Istio's ambient mode with OpenShift Service Mesh, ensure the following prerequisites are met:
+
+* **OpenShift Container Platform 4.19+:** This version of OpenShift is required for supported Kubernetes Gateway API CRDs, which are essential for ambient mode functionalities.
+* **OpenShift Service Mesh 3.1.0+ operator is installed:** Ensure that the OSSM operator version 3.1.0 or later is installed on your OpenShift cluster.
+
+**Pre-existing Service Mesh Installations:**
+
+While the use of properly defined discovery selectors will allow a service mesh to be deployed in ambient mode alongside a service mesh in sidecar mode, this is not a scenario we have thoroughly validated. To avoid potential conflicts, as a technology preview feature, Istio's ambient mode should only be installed on clusters without a pre-existing OpenShift Service Mesh installation.
+
+**Note**: Istio's ambient mode is completely incompatible with clusters containing the OpenShift Service Mesh 2.6 or earlier versions of the operator and they should not be used together.
+
+---
+
+## 3. Procedure to Install Istio's Ambient Mode
+
+This procedure demonstrates how to install Istio's ambient mode on OpenShift with Istio's sample Bookinfo application.
+
+### 3.1 Prerequisites
+
+* You have deployed a cluster on OpenShift Container Platform 4.19 or later.
+* You are logged in to the OpenShift Container Platform we console as a user with the cluster-admin role.
+* The OpenShift Service Mesh operator is [installed](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0/html/installing/ossm-installing-service-mesh#ossm-installing-operator_ossm-about-deploying-istio-using-service-mesh-operator).
+
+### 3.2 Install Istio (Control Plane)
+
+#### 3.2.1 Creating the Istio project using the web console
+
+**Prerequisites**
+
+1.  The Red Hat OpenShift Service Mesh Operator must be installed.
+2.  You are logged in to the OpenShift Container Platform web console as cluster-admin.
+
+**Procedure**
+
+1.  In the OpenShift Container Platform web console, click **Home** > **Projects**.
+2.  Click **Create Project**.
+3.  At the prompt, enter a name for the project in the **Name** field. For example, `istio-system`. The other fields provide supplementary information to the `Istio` resource definition and are optional.
+4.  Click **Create**. The Service Mesh Operator deploys Istio to the project you specified.
+
+#### 3.2.2 Creating the Istio resource using the web console
+
+**Prerequisites**
+
+1.  The Red Hat OpenShift Service Mesh Operator must be installed.
+2.  You are logged in to the OpenShift Container Platform web console as cluster-admin.
+
```

Comment:
These prerequisites are repeated for each of the steps. I think we can have a common pre-requisite section above that applies to all of them.

Reply:
Agree.

I'll restructure this.